shared/tasks/browser.mjs

```diff
@@ -32,17 +32,29 @@ export async function screenshots () {
   const browser = await launch()
   const componentNames = await getDirectories(join(paths.src, 'govuk/components'))
   const exampleNames = ['text-alignment', 'typography']
+  const rebrandExamples = [
+    {
+      component: 'header',
+      example: 'full-width-with-navigation'
+    },
+    {
+      component: 'footer'
+    },
+    {
+      component: 'cookie-banner'
+    }
+  ]
 
   // Screenshot stack
   const input = []
 
   // Add components to screenshot
   input.push(...componentNames.map((screenshotName) =>
-    /** @type {const} */ ([screenshotComponent, screenshotName])))
+    /** @type {const} */([screenshotComponent, screenshotName])))
```

Comment:
Any idea why this has changed? Some sort of auto formatting going on?

Reply:
I think I know why this is. I originally tried to pass the new screenshots into the `input` array and ran into trouble. I suspect this is just a hangover from that. As per Ollie's comment above I'm planning on revisiting this so it may become redundant quite soon.